### PR TITLE
feat: Add typings

### LINKED ISSUE #53: TypeScript typings
👋 @anishkny! Great library, I love it!

I was [experimenting](https://github.com/blockprotocol/blockprotocol/pull/447) with Playwright today and had to add local typings for `playwright-test-coverage`:

```ts
// playwright-test-coverage.d.ts
declare module "playwright-test-coverage" {
  export * from "@playwright/test";
}
```

WDYT of including the typings so that other folks did not have to do this? Writing Playwright tests in TS must be quite common.

**PS:** I initially [opened](https://github.com/mxschmitt/playwright-test-coverage/issues/11) this issue in the wrong repo by mistake. It'd be great if you could also add the [`repository`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#repository) field in `package.json` to avoid similar mistakes in the future.

### PR DESCRIPTION
**Pull request by anishkny**
Closes #53

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module "playwright-test-coverage" {
+  export * from "@playwright/test";
```
Comment (kachkaev) [inline index.d.ts:2]:
```suggestion
  export { expect, test } from "@playwright/test";
```

**Review by kachkaev — COMMENTED** [review]
Hi @anishkny, thanks for this PR! I tried your diff in my experiment (https://github.com/blockprotocol/blockprotocol/pull/447) and it worked!

The only thing I overlooked in https://github.com/anishkny/playwright-test-coverage/issues/53 was the scope of exports. I thought `playwright-test-coverage` was re-exporting everything from `@playwright/test`, but it was actually just `test` and `expect`.

My diff suggestion should do the job.

**Comment by kachkaev** [comment]
We can consider switching back to `export *` later if we re-export everything from `@playwright/test`. This will allow us to do this, for instance:

```ts
import { devices } from "playwright-test-coverage"
```

Direct imports from `@playwright/tests` can be completely banned for consistency. I’d say that this is out of scope for this PR.

**Comment by kachkaev** [comment]
Thanks for addressing my comment @anishkny, the diff LGTM! 👏

**Comment by anishkny** [comment]
Shipped in [v1.1.0](https://www.npmjs.com/package/playwright-test-coverage/v/1.1.0)

**Pull request merged**